Meadowlark-js/backends/meadowlark-opensearch-backend/src/repository/OpenSearchException.ts

```diff
@@ -28,7 +28,11 @@ export async function LogOpenSearchErrors(
       case 'SerializationError':
       case 'TimeoutError': {
         if (openSearchClientError?.message !== undefined) {
-          Logger.debug(`${moduleName} ${documentProcessError}`, traceId, openSearchClientError.message);
+          Logger.debug(
+            `${moduleName} ${documentProcessError}`,
+            traceId,
+            `(${openSearchClientError.name}) - ${openSearchClientError.message}`,
```

Comment:
Please update to `Logger.error` as these are severe problems that should always be  logged.

Reply:
The Logger.debug have been replaced to use the Logger.error.